Manager/PostManager.php

```diff
@@ -114,4 +114,19 @@ public function getThumbnailPath(Post $post)
 
         return $post->getGuid();
     }
+    
+    /**
+     * @param \DateTime $date
+     * @return array
+     */
+    public function findByDate(\DateTime $date)
+    {
+        $qb = $this->repository->createQueryBuilder('c');
```

Comment:
`->setParameter('date', $date->format('Y-m-d') . '%');`should be a bit more efficient :)
Also, the sort should be made by date first, then by ID shouldn't it ?